grpc-api/src/main/java/io/deephaven/grpc_api/session/SessionState.java

```diff
@@ -447,9 +503,12 @@ public T get() {
             if (isExpired()) {
                 throw GrpcUtil.statusRuntimeException(Code.UNAUTHENTICATED, "session has expired");
             }
-            if (state != ExportState.EXPORTED) {
-                throw new IllegalStateException("Dependent export '" + exportId + "' is " + state.name() + " and not " + ExportState.EXPORTED.name() + ".");
+
+            // Note: an export may be released while still being a dependency of queued work; so let's make sure we're still valid
+            if (result == null) {
+                throw new IllegalStateException("Dependent export '" + exportId + "' is " + state.name() + " and not exported.");
```

Comment:
I've lost context: how do we know this is a "dependent" export? And can you delete the trailing '.'?

Reply:
It's a dependency because you're trying to resolve it? If you didn't want it, you shouldn't call get?